packages/shared/src/graphql/sources.ts

```diff
@@ -1,4 +1,18 @@
 import { gql } from 'graphql-request';
+import { UserShortProfile } from '../lib/user';
+import { Connection } from './common';
+
+export enum SourceMemberRole {
+  Member = 'member',
+  Owner = 'owner',
```

Comment:
Maybe already introduce moderator here?

Reply:
Agreed, I already added it in one of my PRs.